src/server/config/schema.js

```diff
@@ -214,4 +214,8 @@ export default () => Joi.object({
     defaultLocale: Joi.string().default('en'),
   }).default(),
 
+  __unstable_plugins: Joi.object({
```

Comment:
__question:__ is there any reason why we can't reuse existing `plugins.scanDirs` config value?

Reply:
Not sure it's possible to re-use as we'll be using it to find plugins in the current platform until K7(?)

Reply:
Ah, I see, so `plugins.scanDirs` will be used for plugins that are handled by legacy platform and `__unstable_plugins.scanDirs` will be used by the new platform plugins, right?

Reply:
Wasn't sure where to put this value (and what to name it). Maybe it needs to live under the current `plugins`, but with something that indicates "this is for the new platform"? Very open to ideas.

Reply:
(we can also discuss in the platform meeting)

Reply:
+1 for dedicated `__unstable` config node. Maybe I'd have something like `__unstable.plugins.scanDirs` to have a room for unstable non-plugins related stuff, but we can do it whenever we need this.

Reply:
++ I'll change it to `__unstable`